mono_repo/lib/src/commands/travis.dart

```diff
@@ -189,17 +189,17 @@ String _travisSh(List<String> tasks, bool prettyAnsi, String pkgVersion) => '''
 #!/bin/bash
 # ${_createdWith(pkgVersion)}
 
-if [ -z "\$PKG" ]; then
+if [[ -z \${PKG} ]]; then
   ${safeEcho(prettyAnsi, red, "PKG environment variable must be set!")}
   exit 1
 fi
 
-if [ "\$#" == "0" ]; then
+if [[ "\$#" == "0" ]]; then
   ${safeEcho(prettyAnsi, red, "At least one task argument must be provided!")}
   exit 1
 fi
 
-pushd \$PKG
+pushd \${PKG} || exit \$?
```

Comment:
The `[[ -z ${PKG} ]]` is probably safe over `[[ -z "${PKG}" ]]`, but I think this should be `pushd "${PKG}" || exit $?`

Reply:
In case spaces?

Reply:
File an issue for me – will follow-up

Reply:
https://github.com/dart-lang/mono_repo/pull/134